### ISSUE TITLE
dumb tag homepages [3]

### ISSUE THREAD

**Issue by aguestuser** [public-accountability/littlesis-rails#234]
# Acceptance Criteria

* when i go to a tag's homepage, i will see 3 lists stacked on the same page in the following order:
  1. a list of entities with the same tag (sorted by and displaying last updated date)
  2. a list of lists with same tag (sorted by date)
  3. a list of relationships with same tag (sorted by date)
* if any list is empty we still see the header, but we see a message explaining why the list is empty
* if any list has more than 10 elements it will be paginated

# Out of Scope

* sort tagged entity list by number of relationships to similarly tagged entities

**Comment by aepyornis** [comment]
blocked by #232

**Comment by aguestuser** [comment]
descoping the bit about "link to tag overview page" from this card. instead, including the insertion of such a link into the tag homepages as an extra AC for #232, where it makes more sense to complete. that eliminates the dependency between these two cards.

**Comment by aguestuser** [comment]
**UPDATE:**

Due to miscommunication (@aguestuser's fault!) on last scrum call, we are committing the heinous Agile sin of adjusting Acceptance Criteria mid-sprint in order to avoid scope creep on this card.

Namely, @aguestuser had proposed that we limit the scope of this card by not grouping tagged items by resource type and instead dumping them all into one list sorted by some interesting criteria. @kconn proposed that this criteria be number of relationships to similarly-tagged items. However, when implementing, we realized that this is an incoherent spec. Namely: the list of tagged items include Entities, Relationships, and Lists. Only Entities have relationships, thus sorting the list by number of relatoinships is incoherent. Additionally, sorting Entities in the manner described is fairly complex (both in terms of correctness and performance) and would require some time to get right.

To keep the scope of the card in check, we are going to:

1. Add the requirement of grouping tagged items by resource type (Entity/Relationship/List) into this card.
2. Extract the requirement of sorting Entities into a separate card to be played later.

While analyzing this card, we also realized that there are likely additional cards to figure out:

1. Grouping subtypes of each resource type
2. How to artfully display separate resource types (likely grouping them under tags like in entity homepages)

We will also make new cards for those enhancements.

---
cc: @aepyornis @kconn
